core/src/com/fallenflame/game/LevelController.java

```diff
@@ -466,8 +476,14 @@ public void update(float dt) {
                 }
             }
 
+            // Update level model.
+            levelModel.update(player, enemies);
+
```

Comment:
This is fine, but please update earlier comments in this method that state "(and update levelModel)"